messenger.rst

```diff
@@ -653,6 +653,20 @@ config and start your workers:
 
 See the `Supervisor docs`_ for more details.
 
+Graceful Shutdown
+~~~~~~~~~~~~~~~~~
+
+Workers can handle the ``SIGTERM`` POSIX signal to finish handling their
+current message before exiting, as long as the `PCNTL`_ extension is installed.
+
+.. tip::
+
+    In some cases the ``SIGTERM`` signal is sent by Supervisor itself (e.g.
+    stopping a Docker container having Supervisor as its entrypoint).
+    In these cases you need to add a ``stopwaitsecs`` key to the program
+    configuration (with a value of the desired grace period in seconds)
+    in order to perform a graceful shutdown.
```

Comment:
```suggestion
    in order to perform a graceful shutdown.
    
    .. code-block:: ini
    
        [program:x]
        stopwaitsecs=20
```
?

Reply:
It usually contains other sections and many more keys inside but yes that's correct. I wouldn't add a longer code example anyways.